server/infra/messaging/rabbit-mq-messaging.js

```diff
@@ -31,4 +32,14 @@ const subscribe = (async (subscriptionName, eventHandler) => {
     }
 });
 
-module.exports = { publish, subscribe };
\ No newline at end of file
+const unsubscribeAll = (async () => {

```

Comment:
Curious, when would we use `unsubscribeAll`?

Reply:
I just used it for test workflow :)